x-pack/legacy/plugins/apm/public/components/app/Settings/AgentConfigurations/AgentConfigurationCreateEdit/SettingsPage/SettingFormRow.tsx

```diff
@@ -93,7 +93,10 @@ function FormRow({
             <EuiFieldNumber
               placeholder={setting.placeholder}
               value={(amount as unknown) as number}
-              min={'min' in setting ? setting.min : 1}
+              // Min and max settings are string in the duration type, representing the amount and unit e.g.: '1s'.
+              // becasue of that only uses it when it's defined as number.
+              min={typeof setting.min === 'number' ? setting.min : undefined}
```

Comment:
`min` should either always be a string or always be a number (I think). When is it defined as a number?

Reply:
For `byte` type, min and max are numbers.
```js
interface BytesSetting extends BaseSetting {
  type: 'bytes';
  min?: number;
  max?: number;
  units?: string[];
}
```

Reply:
Byte should behave like duration I think so `min` is eg. "1kb"